platform/src/main/java/org/hillview/sketches/FullCorrelationSketch.java

```diff
@@ -15,6 +18,7 @@
  * the BasicColStats's.
  */
 public class FullCorrelationSketch implements ISketch<ITable, CorrMatrix> {
+    private static final Logger LOG = Logger.getLogger(FullCorrelationSketch.class.getName());
```

Comment:
I believe that this class will need to be modified since the mean has to be computed across all parts of a dataset.
Talk to Parikshit if it is not clear how to do this in one pass.
The actual correlation matrix will be computed on the root node only; this sketch will compute enough information to enable the computation.